### ISSUE TITLE
Suggestion: Add bar graph between elements

### ISSUE THREAD

**Issue by Toseflo** [carykh/BarrierChain#5]
I really like your idea of a barrier chain, as it shows a nice way how to care about aesthetics while also keeping the academic side of things ;)

One improvement I have for the graph would be: Currently it is quite difficult to estimate the ratio of the values of neighboring cells. 
Adding a bar graph between every two items gives an intuitive view of how big the two items are compared to each other without having to read the numbers.
The graphs also help to understand the direction in which the values are increasing. With just the bars, it might be hard to tell if you are on an ascending or descending column. The bar graphs show this by having one bar larger than the other.

(Values are not correctly scaled)
![Chain](https://github.com/user-attachments/assets/bd5a65c2-a6f5-48de-a1e8-3c9d9cb1f23a)

I also thought about adding lines to make it clearer that the graph above and below each entry shows the same value. But I'm not entirely happy with the result, and maybe someone else has a better idea.

![Chain Lines](https://github.com/user-attachments/assets/bbeafe4b-cfa9-4e67-8de5-45eab6ad5c0b)

**Comment by EwoutH** [comment]
Since the top bar is always full, isn’t it redundant?

**Comment by Toseflo** [comment]
Yes and no. I actually tested this idea with only one bar first. But with only one bar, it felt like it still wouldn't be very intuitive for people who haven't seen a chart like this before. The second bar kinda gives a reference of what to expect, but I guess there is still much room to experiment. 

The other advantage of the second bar is to show the flow direction of the chain. With only one bar, this wouldn't be clear.
